soc/v2/rockchip/rk3399/Kconfig

```diff
@@ -8,3 +8,6 @@ config SOC_RK3399
 	select ARM64
 	select CPU_CORTEX_A53
 	select ARM_ARCH_TIMER if SYS_CLOCK_EXISTS
+
+config SOC_PART_NUMBER
+	default "RK3399" if SOC_SERIES_RK3399
```

Comment:
Why would this be here and excluded from sysbuild/twister usage?

Reply:
because `SOC_PART_NUMBER` is defined here:
https://github.com/zephyrproject-rtos/zephyr/blob/5c03689a1ed563aeff0feccd13eba444b9e95959/soc/arm/Kconfig#L48-L49

which is only sourced in Zephyr's Kconfig tree, and not in the self-contained hwm v2 Kconfig tree.


We could consider if the `arm/Kconfig` file and similar files should be sourced into the self-contained tree, but then we need to look at their `select` which are pointing into general Zephyr Kconfig.
Alternative, split the file in two, as we have done for other Kconfig.

I'm fine with doing a split on the referenced file.

Reply:
let's leave it as-is for now, can consider it a future improvement if it's needed